extensions/mssql/package.json

```diff
@@ -404,6 +426,23 @@
               "displayName": "%cloud.serverProperties.serverEdition%",
               "value": "serverEdition"
             }
+          ],
+          "databasesListProperties": [
+            {
+              "displayName": "%databasesListProperties.name%",
+              "value": "name",
+              "widthWeight": 60
+            },
+            {
+              "displayName": "%databasesListProperties.state%",
```

Comment:
Portal and mockups have this column as "Status" instead of "State"

Reply:
i will change the display name. the field name is coming from STS, i'll leave it as state.